plug_pass_prototype_0_1.ino

```diff
@@ -180,61 +190,12 @@ void loop () {
     // Wait a bit before trying again
     Serial.println("\n\nSend a character to scan another tag!");
     Serial.flush();
-   // while (!Serial.available());
+    while (!Serial.available());

```

Comment:
this while loop was preventing the code from moving on.  was that on purpose?

Reply:
It appears it was supposed to be an infinite loop to delay until serial became available. I don't know further.

Reply:
what's it mean for serial to become available?